### PR TITLE
About: print information about squashing of inductives

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: vernac/prettyp.ml [hunk 1]
```diff
@@ -208,6 +208,54 @@ let print_polymorphism env ref =
...
+          | Type _ -> str "not in a variable sort quality"
+          | QSort (q,_) -> str "in sort quality " ++ Termops.pr_evd_qvar sigma q
+      in
+      let unless = match squash with
+        | AlwaysSquashed -> str "."
+        | SometimesSquashed qs ->
+          let target = match inds with
+            | SProp | Prop | Set -> target
+            | Type _ -> str "instantiated to constant qualities"
+            | QSort (q,_) -> str "smaller than the instantiation of " ++ Termops.pr_evd_qvar sigma q
```
Comment (ppedrot) [inline vernac/prettyp.ml:240]: "smaller" is confusing here. I think we should use another word, like "eliminable" or something like it.

Comment (SkySkimmer) [reply vernac/prettyp.ml:240]: That's not defined anywhere though I think

File: vernac/prettyp.ml [hunk 2]
```diff
@@ -208,6 +208,57 @@ let print_polymorphism env ref =
...
+      let unless = match squash with
+        | AlwaysSquashed -> str "."
+        | SometimesSquashed qs ->
+          let target = match inds with
+            | SProp | Prop | Set -> target
+            | Type _ -> str "instantiated to constant qualities"
+            | QSort (q,_) ->
+              let ppq = Termops.pr_evd_qvar sigma q in
+              str "equal to the instantiation of " ++ ppq ++ pr_comma() ++
+              str "or to Prop or Type if " ++ ppq ++ str " is instantiated to Type"
```
Comment (SkySkimmer) [inline vernac/prettyp.ml:243]: how about like this?

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot run full ci

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**